### PR TITLE
Rs 2022 04 11

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 1]
```diff
@@ -221,19 +227,19 @@ I get off something resembling a retort before averting my gaze from the merry A
...
+ the park would be full of young couples and students on their way back from a night out,
+ even at this hour.
 }
 [sha:dc4e537d204ceb836250f1a6c5611de16cc343f6]{
 -- Page 575, Offset 25754.
 -- 今では、ここで話しているのは自分とアルクェイドだけ。
-// is this ここで(話しているのは自分とアルクェイド)だけ
-// or (ここで話している)のは自分とアルクェイドだけ
-Now the only people here are me and Arcueid, chatting.
+But the only people here chatting now are myself and Arcueid.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:237]: I actually prefer the previous interpretation, as did Manolo fwiw.

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 2]
```diff
@@ -479,12 +490,12 @@ Arcueid takes a deep breath before putting on a forced smile.
...
 -- Page 591, Offset 25804.
 -- 「そうだよねー。わたしもホントはそう思ってたんだ。
-"I guess you're right. I've thought the same.
+"I guess you're right. I've actually thought the same.
 }
 [sha:c7b4e05f43ec3155ccda989ec205d2a24b3db82f]{
 -- Page 591, Offset 25805.
 -- 果たして、アルクェイド・ブリュンスタッドは本当に吸血鬼なのでしょうか？　って」
-I've thought to myself, is Arcueid Brunestud really a vampire?"
+I've wondered to myself, is Arcueid Brunestud really a vampire?"
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:498]: I've wondered to myself: 'is Arcueid Brunestud really a vampire?'"

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 3]
```diff
@@ -589,47 +601,49 @@ Arcueid is getting closer.
...
 -- Page 599, Offset 25829.
 -- 「志貴は、わたしのことを半人前だって言ったけど」
-"You said that I was only a half-vampire."
+"You said that I was barely half a vampire."
 }
 [sha:0b56353a1ddf1fd4f7f98b22dcb22d9097d9df32]{
 -- Page 599, Offset 25830.
 -- 頭の後ろに響くような、甘い声。
-An alluring voice that rings in the back of my head.
+Her alluring voice resounds inside my head.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:626]: reverberates

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 4]
```diff
@@ -814,12 +833,13 @@ It's only Arcueid who's trembling violently.
...
 -- 苦しげな呼吸。
-An agonizing breath.
+Her breathing is pained.
 }
 [sha:137b914d38d6306820187f074d5e659675606d4d]{
 -- Page 609, Offset 25872.
 -- アルクェイドの息遣いはこんなにも近いのに、遥か遠くのものに感じる。
-Even though her breath is so close, it feels so far away.
+// Mild liberties to avoid a dumb sounding repetition of breathing
+Even though each breath is so close, they feel so far away.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:842]: is so close, it feels so far away

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt [hunk 5]
```diff
@@ -1120,15 +1148,15 @@ I let out a scream.
 [sha:4cc28a6401e9ae67b92e2b675a1125d9653361c4]{
 -- Page 624, Offset 25932.
 -- 自分で、自分の声を聞いて情けないな、と思った瞬間。
-In the moment when I thought to myself how pathetic it is to hear my own voice.
+Hearing my own voice like that, it sounded almost pathetic for a moment.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5D.txt:1151]: for a moment, it sounded

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:237]: Hmm, the left-associativity makes more sense to me with the のは. I'll phone a friend.

File: script/Arcueid/Day 10/10_00_ARC10_5D.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:842]: Hm I think plural is correct here? There are multiple breaths

Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_5D.txt:842]: Although I'm not so sure I agree with my reasoning anymore, I do agree with my assessment that something was off. I say reallocate the "each" to the second bit.

> Even though they're so close, each breath feels so far away.

**Pull request merged**